### PR TITLE
43 update styles for home screen

### PR DESCRIPTION
**Pull request by carlosgonzalez96**

### CODE REVIEW

File: src/components/Groups/CreateGroupButton.tsx [hunk 1]
```diff
@@ -2,21 +2,28 @@ import React, { Component } from 'react';
...
         return (
-            <Button onPress={this.handlePress}>
+            <Button onPress={this.handlePress} style={styles.createGroupb}>
                 <ButtonText>Create Group</ButtonText>
             </Button>
         );
     }
 }
 
+const styles = StyleSheet.create({
```
Comment (shamblid) [inline src/components/Groups/CreateGroupButton.tsx]: Lets try to get these style sheets into styled-components. Look at the other components for how to do this.

File: src/components/StreakImage.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+import {
+    View
+} from 'native-base';
+
+import {
+    Streak
+} from '../components'
+
+export class StreakImage extends React.Component {
```
Comment (shamblid) [inline src/components/StreakImage.tsx]: When we aren't managing state usually we want a pure component for optimization. So in this case I would make this class into a function component. And even if we need to manage state, we can use react hooks.

**Comment by carlosgonzalez96** [comment]
The real review should be seen in the styling folder under component and the StreakImage.tsx file.

File: src/components/styling/StreakImageStyles2.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+export const View = styled.View`
+    justify-content: center;
+    align-items: center;
+`;
+
+export const FlameImgBg = styled.ImageBackground`
+    width: 100px;
+    height: 143px;
+    position: relative;
+    textAlign: center;
```
Comment (shamblid) [inline src/components/styling/StreakImageStyles2.tsx]: Think you may have an error with this one which should be `text-align`

File: src/components/Groups/CreateGroup/CreateGroupFormSyles.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { StyleSheet } from 'react-native';
```
Comment (shamblid) [inline src/components/Groups/CreateGroup/CreateGroupFormSyles.js]: I would remove this file and pass in a background color prop to the styled component. 

Would look something like this https://www.styled-components.com/docs/basics#passed-props

File: src/components/Groups/JoinGroup/JoinGroupForm.tsx [hunk 5]
```diff
@@ -2,12 +2,19 @@ import React from 'react';
 import { Button, ButtonText } from '../../basic';
 import { Container } from './join_group_styles';
 import Picker from './Picker';
+import { StyleSheet } from 'react-native'
 
 export const JoinGroupForm = props => (
     <Container>
         <Picker {...props} />
-        <Button onPress={props.handleSubmit} disabled={!props.isValid}>
+        <Button onPress={props.handleSubmit} disabled={!props.isValid} style={styles.JoinGroupButtonStyle}>
```
Comment (shamblid) [inline src/components/Groups/JoinGroup/JoinGroupForm.tsx]: Same with this remove the style prop and pass in `backgroundColor` prop which you can access in the button.

File: src/screens/SettingsScreen.tsx [hunk 6]
```diff
@@ -1,12 +1,32 @@
...
+      <Container style={styles.container}>
+        <Text>Nice</Text>
+        <SignoutButton />
+      </Container>
+
+    );
   }
 }
+
+const styles = StyleSheet.create({
```
Comment (shamblid) [inline src/screens/SettingsScreen.tsx]: Can we make this into a styled component as well?

File: src/screens/SettingsScreen.tsx [hunk 7]
```diff
@@ -1,12 +1,32 @@
 import React from 'react';
-import { Settings } from '../components';
-import { Text } from 'react-native';
+import { Settings, SignoutButton } from '../components';
+import { Text, StyleSheet } from 'react-native';
+import Constants from 'expo-constants';
+import {
+  Container, View
+} from 'native-base';
```
Comment (shamblid) [inline src/screens/SettingsScreen.tsx]: Lets not use native-base here and create our own components

File: src/screens/SettingsScreen.tsx [hunk 8]
```diff
@@ -1,12 +1,21 @@
 import React from 'react';
-import { Settings } from '../components';
+import { Settings, SignoutButton } from '../components';
 import { Text } from 'react-native';
+import Constants from 'expo-constants';
+import { Container } from '../components/styling/SettinngsScreenStyles';
+import { from } from 'zen-observable';
```
Comment (shamblid) [inline src/screens/SettingsScreen.tsx]: don't think this is being used

Also the settings styling component file is not grammatically correct. Lets fix that.

**Pull request merged**